.gitignore

```diff
@@ -133,5 +133,9 @@ dmypy.json
 .idea
 .vscode
 
+# comma seperated vals report files
+*.csv
+!tests/**/*_example.csv
+
```

Comment:
@tzaffi Friendly pointer:  For tests moved to `utils/test/teal/app_utest_noop_args.teal`, I think there's a missing a `.gitignore` entry.  The previous entry (`tests/**/*.teal`) no longer covers all scenarios.

Reply:
@tzaffi Thanks - #297 addresses the concern.  I'm assuming we'll merge #297 in here.  Resolving.